examples/wifi-plotting-xarray.py

```diff
@@ -57,12 +58,19 @@
 ###############################
 
 
-def get_average_throughput(stdout):
-    # This function takes a string and parses it to extract relevant
-    # information
-    m = re.match('.*throughput: [-+]?([0-9]*\.?[0-9]+).*', stdout,
-                 re.DOTALL).group(1)
-    return float(m)
+def get_average_throughput(result):
+    # This function takes a result and parses its standard output to extract
+    # relevant information
+    stdout_file = campaign.db.get_result_files(result['id'])['stdout']
+    stderr_file = campaign.db.get_result_files(result['id'])['stderr']
+    with open(stderr_file, 'r') as stderr:
+        if stderr.read() != '':
+            print(stderr.read())
+    with open(stdout_file, 'r') as stdout:
+        stdout = stdout.read()
+        m = re.match('.*throughput: [-+]?([0-9]*\.?[0-9]+).*', stdout,
+                     re.DOTALL).group(1)
+        return float(m)
 
 
 # Reduce multiple runs to a single value (or tuple)
```

Comment:
It seems that you remove the statistic function here and let users to define how to calculate stats data from the results. I think this change is reasonable because we try to avoid making this model as a statistic one.